src/assets/scss/carbon-override.scss

```diff
@@ -1,4 +0,0 @@
-// forces the navigation items to be displayed at all viewport sizes
```

Comment:
Keeping these files in scss could provide a useful structure for project organization, even if they aren't particularly useful in an empty project.

Is there something else that the starter is proposing instead?